x-pack/plugins/task_manager/server/routes/metrics.ts

```diff
@@ -39,8 +40,11 @@ const QuerySchema = schema.object({
 export function metricsRoute(params: MetricsRouteParams) {
   const { router, logger, metrics$, resetMetrics$, taskManagerId } = params;
 
-  const debugLogger = logger.get(`metrics-debugger`);
-  const additionalDebugLogger = logger.get(`metrics-subscribe-debugger`);
```

Comment:
don't have to do it in this PR but we can remove this metrics debugger logs!

Reply:
Might as well! https://github.com/elastic/kibana/pull/192574/commits/14881396f50ebe594904ddd467408e4fc0c500a3